src/dcat-us/dataset-formatter.test.ts

```diff
@@ -977,11 +975,275 @@ describe('formatDcatDataset', () => {
       },
     };
 
-    try {
-      const formatted = JSON.parse(formatDcatDataset(dataset, siteUrl, siteModel, buildDatasetTemplate()));
-      expect(formatted.spatial).toEqual('');
-    } catch {
-      fail('Should not throw!');
-    }
+    const formatted = JSON.parse(formatDcatDataset(dataset, siteUrl, siteModel, buildDatasetTemplate({ spatial: undefined })));
+    expect(formatted.hasOwnProperty('spatial')).toBeFalsy();
```

Comment:
The test title says `does not define spatial property`, so perhaps we should check with `.toBeUndefined()` instead of `.toBeFalsy()`to be more explicit.

Reply:
And if this is something you change, we'll want to change it in all the other tests too

Reply:
In this case I am actually being more specific.

The test should **fail** if spatial is ever returned with an `undefined` value. In that case, `expect(format.spatial).toBeUndefined()` does not actually assert what we want, as `{ spatial: undefined }` would pass the test when it should fail. Instead, each dataset should **not** have the property at all, hence why I went with hasOwnProperty.

Reply:
🤦‍♂️ Sorry, the `hasOwnProperty` didn't register in my mind until you pointed it out. You are good to go.